packages/medusa/src/api-v2/admin/price-lists/[id]/route.ts

```diff
@@ -29,3 +31,77 @@ export const GET = async (
 
   res.status(200).json({ price_list: priceList })
 }
+
+export const POST = async (
+  req: AuthenticatedMedusaRequest<AdminPostPriceListsPriceListReq>,
+  res: MedusaResponse
+) => {
+  const id = req.params.id
+  const workflow = updatePriceListsWorkflow(req.scope)
+
+  const [_, count] = await listPriceLists({
+    container: req.scope,
+    fields: ["id"],
+    variables: { filters: { id }, skip: 0, take: 1 },
+  })
+
+  if (count === 0) {
+    throw new MedusaError(
+      MedusaError.Types.NOT_FOUND,
+      `Price list with id: ${id} was not found`
+    )
+  }
+
+  const { errors } = await workflow.run({
+    input: { priceListsData: [{ id, ...req.validatedBody }] },
+    throwOnError: false,
+  })
+
+  if (Array.isArray(errors) && errors[0]) {
+    throw errors[0].error
+  }
+
+  const [[priceList]] = await listPriceLists({
+    container: req.scope,
+    fields: defaultAdminPriceListFields,
+    variables: { filters: { id }, skip: 0, take: 1 },
+  })
+
+  res.status(200).json({ price_list: priceList })
+}
+
+export const DELETE = async (
+  req: AuthenticatedMedusaRequest,
+  res: MedusaResponse
+) => {
+  const id = req.params.id
+  const workflow = deletePriceListsWorkflow(req.scope)
+
+  const [_, count] = await listPriceLists({
+    container: req.scope,
+    fields: ["id"],
+    variables: { filters: { id }, skip: 0, take: 1 },
+  })
+
+  if (count === 0) {
```

Comment:
todo: Should be idempotent, just return:
```
{
    id,
    object: "price_list",
    deleted: true,
  }
```

...even though the ID didn't exist.